crates/bevy_math/src/rotation2d.rs

```diff
@@ -0,0 +1,332 @@
+use glam::FloatExt;
+
+use crate::prelude::{Direction2d, Mat2, Vec2};
+
+/// A counterclockwise 2D rotation in radians.
+///
+/// The rotation angle is wrapped to be within the `]-pi, pi]` range.
```

Comment:
I don't think so, this is a common way to describe inclusion or not of the range boundaries. (open or close range boundaries).
Here  `]-pi,pi]` explicit states all values withn `-pi` `pi`, excluding `-pi` itself, but including `pi`.

Reply:
Ah ok, I'm used to `(-pi, pi]`, but if it's common to do it the way stated then TIL and nvm the suggestion.

Reply:
I had never seen this before. Strongly prefer standard interval notation here.

Reply:
Fwiw, Nalgebra also uses this: https://docs.rs/nalgebra/latest/nalgebra/geometry/type.UnitComplex.html#method.angle

Reply:
This isn't a hill I'm willing to die on, but I do think the engine should pick one or the other for consistency. In absence of a style guide I suspect the anglophone notation is already more prevalent in the engine docs, but I will have to check.

Reply:
Both notations are valid, but it looks like the parenthesis version is generally more common, yeah. The main benefit of the reverse square bracket is that it avoids misunderstandings where one might think e.g. `(x, y)` is a point or some other pair of objects `x` and `y` instead of an open interval. But that's not an issue here, so I'm fine changing it to `(-pi, pi]`.

Reply:
Changed it to the parenthesis notation